app/pages/lab/collaborators.cjsx

```diff
@@ -84,6 +95,10 @@ CollaboratorCreator = createReactClass
   showTranslatorRole: ->
     if (@props.project.experimental_tools?.indexOf('translator-role') > -1) or isAdmin()
       POSSIBLE_ROLES = Object.assign({}, POSSIBLE_ROLES, {translator: 'translator'});
+  
+  showMuseumRole: ->
+    if (@props.project.experimental_tools?.indexOf('museum-role') > -1) or isAdmin()
+      POSSIBLE_ROLES = Object.assign({}, POSSIBLE_ROLES, {museum: 'museum'});
```

Comment:
Should the `POSSIBLE_ROLES` have the museum mode setup always in code? 

Instead we could rely on presentation in the UI to only show this `museum` role checkbox if the experimental tool is enabled?

Reply:
> Should the POSSIBLE_ROLES have the museum mode setup always in code?

Sorry Cam, I need a bit more clarification on what you mean?

Currently, the `'museum'` role is only added to `POSSIBLE_ROLES` - and therefore only visible on the UI - when the `museum-role` experimental tool is added to the Project.

Viewing the relevant code...

```
POSSIBLE_ROLES = {
  collaborator: 'admin',
  expert: 'team',
  scientist: 'scientist',
  moderator: 'moderator',
  tester: 'team'
}
```

...the Collaborators page starts off knowing 5 possible roles, then adds `translator` and `museum` only depending on the Project's experimental-tools config.

Reply:
I was thinking it might be easier to have the full set of possible roles already setup instead of modifying that object conditionally later in the code. If that's the way the translator role is setup then this is fine.

Reply:
AH! Okay, big thanks to @wgranger for clarifying this. I understand now that the comment was to _"add the 'museum role' checkbox to the Experimental Features list, on the **Project Admin** page"_

Gotcha! That's a legit change to add to this PR, as I _completely blanked out_ on the fact that the Project Admin page is how normal people (who don't dick around with the zooAPI as their default method of doing things) usually add experimental features.

👌 Give me a sec!